docs/guides/cases/connect-lwm2m.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Connect a LwM2M Device
+sidebar_position: 6
+---
+
+# Connect a LwM2M Device
+
+This document demonstrates the integration of LwM2M protocol devices with Shifu. Shifu communicates with the LeShan server via the LwM2M protocol, showcasing seamless device management and interaction.
+
+
+
+## Introduction to LwM2M protocol
+
+`LwM2M` (Lightweight M2M)is a lightweight IoT device management protocol developed by OMA SpecWorks (Open Mobile Alliance). It is specifically designed for resource-constrained devices, such as low-power sensors and embedded devices, providing remote management capabilities for device management and services. LwM2M operates on the CoAP (Constrained Application Protocol) protocol stack and uses UDP or DTLS as the transport layer protocol, making it suitable for low-bandwidth and unstable network environments.
+
+The following section describes how to use ***Shifu*** to connect devices via the `LwM2M` protocol.
+
+
+
+## Pull and Deploy the Shifu Project
+
+Before pulling the project from GitHub, you need to download the relevant Git environment, which can be quickly installed using yum. After setting up Git, clone Shifu to your local machine. 
+```shell
+sudo yum -y install git && git clone https://github.com/Edgenesis/shifu.git
+```
+Finally, please use the latest configuration file to ensure support for the latest protocols.
+```shell
+kubectl apply -f pkg/k8s/crd/install/shifu_install.yml
+```
```

Comment:
Let's use the production deployment command or jump to the . /install/install-shifu-prod page instead.  In fact, I don't think we need to tell the user to install shifu here.